ashlar/views.py

```diff
@@ -71,6 +73,16 @@ class BoundaryViewSet(viewsets.ModelViewSet):
     serializer_class = BoundarySerializer
     filter_class = BoundaryFilter
 
+    def create(self, request, *args, **kwargs):
```

Comment:
Not sure I understand why this is here ATM. There is no way this would fail, since uniqueness is on an auto generated primary key for this field. (And I kind of assumed the rest framework would throw this error already for uniqueness constraint failures?)